modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replicator/PartitionReplicaListener.java

```diff
@@ -2528,7 +2534,9 @@ private CompletableFuture<CompletableFuture<?>> applyUpdateCommand(
                             true,
                             null,
                             null,
-                            null);
+                            null,
+                            indexIdsAtRwTxBeginTs(catalogService, txId, tableId())
```

Comment:
It looks like all `handleUpdate`/`handleUpdateAll` invocations get the same last argument: `indexIdsAtRwTxBeginTs(catalogService, txId, tableId()`.

I suggest to calculate this inside them and remove it as an argument from these 2 methods.

Reply:
fix it